.github/pull_request_template.md

```diff
@@ -1,8 +1,8 @@
 ### Pull Request checklist ###
 <!-- Before submitting the PR, please address each item -->
 - [ ] **Quality**: This PR builds and tests run cleanly
-  - `cargo clean; cargo test --all` runs without emitting any warnings
-  - `cargo fmt` does not produce any changes to the code
+  - `make test` runs without emitting any warnings
```

Comment:
`make` is not generally available on Windows, or, at least, it's not generally added to the `PATH`, so this won't work. I don't think it's too much of a big deal.

Reply:
Yeah -- I've seen other projects include a `.bat` file for this kind of thing, but then we'd have two things to maintain which is also not ideal.

I'm fine with leaving as-is for now until we have more Windows contributors ;)